action.yml

```diff
@@ -46,6 +55,8 @@ runs:
     - ${{ inputs.tag }}
     - ${{ inputs.date }}
     - ${{ inputs.referenceUrl }}
+    - ${{ inputs.checksSettingsFile }}
+    - ${{ inputs.proLicenseKey }}
```

Comment:
It looks like `hubApiKey` should be here. Am I just missing it somewhere else?

Reply:
Good catch. I did miss it here. I haven't been able to actually test this yet.